packages/aws-rfdk/lib/deadline/scripts/bash/configureWorker.sh

```diff
@@ -36,6 +36,26 @@ if [ ! -f "$DEADLINE_COMMAND" ]; then
     exit 1
 fi
 
+isVersionLessThan() {
+    local RESULT=$(python -c "import sys;print(tuple(map(int,\"$1\".split('.'))) < tuple(map(int, \"$2\".split('.'))))")
+    case $RESULT in
+        True) echo true ;;
+        False) echo false ;;
+        *) echo "ERROR: Unknown boolean value \"$RESULT\"" 1>&2; exit 1 ;;
+    esac
```

Comment:
Suggestion:

1.  Rather than parsing stdout, could we just use the exit-code?
1.  Rather than inlining the versions operands into the python script, can we pass them as arguments? This will help avoid escaping issues.

Putting that together:

```suggestion
    python -c "import sys;sys.exit(0 if tuple(map(int, sys.argv[-2].split('.'))) < tuple(map(int, sys.argv[-1].split('.'))) else 1)" "$1" "$2"
```

using this function would become:

```bash
if isVersionLessThan 10.1.9 10.1.10; then
  echo "less than!"
fi
```

Reply:
updated.